storybook/main.js

```diff
@@ -35,22 +26,51 @@ export default {
         '@storybook/addon-a11y',
         // "@storybook/addon-storysource"
     ],
-    nodeResolve: {
-        exportConditions: ['browser', 'development'],
+    framework: {
+        name: 'storybook-web-components-wds',
     },
-
-    rollupConfig(config) {
+    wdsFinal(config) {
+        const json = fromRollup(rollupJson);
+        return mergeConfigs(config, {
+            nodeResolve: {
+                exportConditions: ['browser', 'development'],
+                moduleDirectories: [
+                    'node_modules',
+                    'packages',
+                    'projects',
+                    'tools',
+                ],
+            },
+            clearTerminalOnReload: false,
+            mimeTypes: {
+                '**/*.json': 'js',
+            },
```

Comment:
I had to disable watch mode here, will investigate together with HMR
for now the server is run in middleware mode as other builder, so watch mode will require a bit different approach